tomee/tomee-microprofile/mp-common/pom.xml

```diff
@@ -158,6 +158,12 @@
       </exclusions>
     </dependency>
 
+    <dependency> <!-- Added in for backwards compatibility with the move to Jakarta EE coordinates -->
```

Comment:
looks like you just want to copy the same than in openejb-core for consistency + use provided scope since on j8 it must not be transitive

see https://github.com/apache/tomee/blob/master/container/openejb-core/pom.xml#L558